adds a root changelog (#4009)
firebase/firebase-js-sdk: CHANGELOG.md

# Changelog

You can view the changelog for the packages within this monorepo in `packages/<package name>/CHANGELOG.md`. For example, you will find the changelog for the Firestore package at [`packages/firestore/CHANGELOG.md`](./packages/firestore/CHANGELOG.md).

> Additionally, you can view a changelog for the entire monorepo at the [firebase support page](https://firebase.google.com/support/release-notes/js).

Reviewer: ```suggestion
Additionally, you can view our official release notes for the entire monorepo at the [firebase support page](https://firebase.google.com/support/release-notes/js).
```
